mern-ecommerce/controllers/auth.js

```diff
@@ -62,20 +62,20 @@ exports.requireSignin = expressJwt({
   userProperty: 'auth',
 });
 
-exports.isAuth = (req, res, next) => {
-  let user = req.profile && req.auth && req.profile._id == req.auth._id;
-  if (!user) {
+exports.isAdmin = (req, res, next) => {
```

Comment:
The `isAdmin` function is checking if the user's role is 0, but it's not clear what role 0 represents. It would be better to define a constant for the admin role and use that instead of a magic number. This would make the code more readable and maintainable.

Additionally, the error message 'Admin resource!' is incomplete. It would be better to provide a more descriptive error message, such as 'Access denied. You are not an admin.'